aiobotocore/endpoint.py

```diff
@@ -51,43 +69,43 @@ def __dir__(self):
         return attrs
 
     def __del__(self):
-        self.close()
+        # callers may not read the Body resulting in an unclosed ClientResponse
+        if self._loop.is_running():
+            asyncio.ensure_future(self.__response.release(), loop=self._loop)
```

Comment:
If body is not fetched, connection and response should be closed with warning like aiohttp does, you can use:
`(self.__response.close()` without `ensure_future`. ensure_future is dangerous here because we can loose exception.